controllers/candidate.js

```diff
@@ -17,13 +17,13 @@ async function getCandidates(req, res, next) {
     }
 }
 
-async function publishCandidate(name, party) {
+async function publishCandidate(objectId, name, party) {
     const election = getElection();
     if (election === null) {
         throw new Error('Election has not started');
     }
 
-    if (!name || !party) {
+    if (!objectId || !name || !party) {
         throw new Error('Candidate name and party are required');
```

Comment:
'Candidate object ID, name and party are required' instead of just name and party

Reply:
Fixed in be58054